.eslintrc.js

```diff
@@ -49,5 +71,13 @@ module.exports = {
       }
     ],
     'vue/require-prop-types': ['error']
+  },
+  settings: {
+    jsdoc: {
+      tagNamePreference: {
+        property: 'prop',
+        returns: 'return'
```

Comment:
Linter defaults to using the old school `@property` and `@returns`, this changes it to accept what most modern JSDoc tools produce by default now, `@prop` and `@return`.